anim.js

```diff
@@ -1,81 +1,82 @@
-"use strict"
+var ease = require('./easing.js').ease;
```

Comment:
Yeah I think you're right. There is also a linting rule to prevent `var` in favor of `const` and `let`.
I'll take a look at it and remove `var` declarations completeley.